.github/workflows/regression-report.yml

```diff
@@ -301,6 +302,14 @@ jobs:
           path: ${{ env.GENERATED_CONFIGS_PATH }}
           key: ${{ env.CACHE_KEY_GENERATED_CONFIGS }}
 
+      - name: Extract filename and create label
+        id: create_label
+        run: |
+          INPUT_FILE="${{ needs.parse_comment.outputs.input_file }}"
+          FILENAME=$(basename "$INPUT_FILE" .java)
```

Comment:
`FILENAME=$(basename "$INPUT_FILE")`
to keep extension of file, it will not always be java

```
$ INPUT_FILE=src/test/resources/com/puppycrawl/tools/checkstyle/checks/annotation/annotationusestyle/InputAnnotationUseStyleWithParens.java
$ FILENAME=$(basename "$INPUT_FILE" .java)
$ echo $FILENAME
InputAnnotationUseStyleWithParens
$ FILENAME=$(basename "$INPUT_FILE")
$ echo $FILENAME
InputAnnotationUseStyleWithParens.java
```

Reply:
updated